packages/keyring-internal-api/src/eth/EthKeyring.ts

```diff
@@ -1,12 +1,11 @@
-import type { Json, Keyring } from '@metamask/utils';
```

Comment:
I considered this `EthKeyring` type to be very internal. It has no real use with the `keyring-api` itself.